src/views/Accessibility/AccessibilityRequestDetailPage.tsx

```diff
@@ -147,11 +157,16 @@ const AccessibilityRequestDetailPage = () => {
           shouldSendEmail: values.shouldSendEmail
         }
       }
-    }).then(() => {
-      refetch();
-      showMessage(''); // allows screen reader to hear consecutive success message
-      showMessage(t('requestDetails.notes.confirmation', { requestName }));
-      resetForm({});
+    }).then(response => {
+      const userErrors =
+        response.data?.createAccessibilityRequestNote?.userErrors;
```

Comment:
@christopherhuii - I had previously handled if there were userErrors here and looks like you removed it.  Is there any reason not to handle them?